project-base/storefront/components/Blocks/Product/LastVisitedProducts/LastVisitedProducts.tsx

```diff
@@ -0,0 +1,29 @@
+import { Heading } from 'components/Basic/Heading/Heading';
+import { ProductsSlider } from 'components/Blocks/Product/ProductsSlider';
+import { GetLastVisitedProductCatalogNumbers } from 'helpers/products/LastVisitedProducts';
+import { Webline } from 'components/Layout/Webline/Webline';
+import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslationFunction';
+import { GtmProductListNameType } from 'types/gtm/enums';
+import { useProductsByCatnumsApi } from 'graphql/generated';
+
+const LastVisitedProducts: FC = () => {
+    const lastVisitedProductCatalogNumbers = GetLastVisitedProductCatalogNumbers();
+    const [result] = useProductsByCatnumsApi({ variables: { catnums: lastVisitedProductCatalogNumbers } });
+    const lastVisitedProducts = result.data?.productsByCatnums ?? [];
+    const t = useTypedTranslationFunction();
+    if (lastVisitedProducts.length !== 0) {
+        return (
+            <Webline className="my-6">
+                <Heading type="h2">{t('Last visited products')}</Heading>
+                <ProductsSlider
+                    products={lastVisitedProducts}
+                    gtmProductListName={GtmProductListNameType.last_visited_products}
+                />
+            </Webline>
+        );
+    }
+
+    return null;
```

Comment:
I would say turning this condition around is more readable and is the standard we are trying to follow.

Reply:
I dont know what you mean ); Please give me more info (;

Reply:
Fixed, check 2 last commits